google/cloud/spanner/keys_test.cc

```diff
@@ -78,6 +78,46 @@ TEST(KeyrangeTest, Accessors) {
       std::is_same<EndType&&, decltype(std::move(range).start())>::value, "");
 }
 
+TEST(KeySetTest, EqualityEmpty) {
```

Comment:
Were these tests moved? I think their old location was preferable, because then they'll come after KeyRangeTest, which matches the order they are defined in the header.